dingomata/cogs/text.py

```diff
@@ -281,14 +286,20 @@ async def waffle(self, ctx: discord.ApplicationContext) -> None:
 
     @slash(cooldown=True)
     @discord.option('user', description="Who to brush")
-    async def brush(self, ctx: discord.ApplicationContext, user: discord.User) -> None:
+    @discord.option('Body Part', description="What to brush", choices=["Head", "Ears", "Tail", "Paws", "Arms"])
+    async def brush(self, ctx: discord.ApplicationContext, user: discord.User, bodyPart: Optional[str],) -> None:
         """Give someone a nice brushing!"""
         if ctx.author == user:
             await ctx.respond(f"{ctx.author.display_name} brushes themselves... Got to look your best!")
         else:
-            await self._post_random_reply(
-                ctx, "brush", target=mention_if_needed(ctx, user),
-                post="Ahhhhhh. That feels nice, thank you!" if user == self._bot_for(ctx.guild.id).user else "")
+            if bodyPart is not None:
+                await self._post_random_reply_with_specific_bodyPart(
+                    ctx, "brush", bodyPart=bodyPart, target=mention_if_needed(ctx, user),
+                    post="Ahhhhhh. That feels nice, thank you!" if user == self._bot_for(ctx.guild.id).user else "")
+            else:
+                await self._post_random_reply(
+                    ctx, "brush", target=mention_if_needed(ctx, user),
+                    post="Ahhhhhh. That feels nice, thank you!" if user == self._bot_for(ctx.guild.id).user else "")
```

Comment:
We probably also don't need to distinguish them with the if-else: if we edit the render function to also ignore keys that are None, we can simply pass it through and it'll fall back to random choice automatically

Reply:
I agree, I wasn't sure how flexible the the format function was to having things in kwargs that weren't referenced in the fragment